patients_core/migrations/0001_initial.py

```
@@ -0,0 +1,33 @@
+            name='Patient',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('national_id', models.CharField(blank=True, max_length=20, null=True, unique=True)),
+                ('full_name', models.CharField(max_length=120)),
+                ('dob', models.DateField(blank=True, null=True)),
+                ('sex', models.CharField(blank=True, choices=[('MALE', 'Male'), ('FEMALE', 'Female'), ('OTHER', 'Other')], max_length=10, null=True)),
+                ('primary_doctor', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, related_name='patients', to=settings.AUTH_USER_MODEL)),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+            ],
+            options={
+                'indexes': ["models.Index(fields=['full_name'], name='patients_co_full_na_6e1f6a_idx')", "models.Index(fields=['created_at'], name='patients_co_created_9d0896_idx')"],
```

Ruff (0.12.2)

Line too long (183 > 88)

(E501)